Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -221,6 +221,70 @@ INT_VECT EmbedMultipleConfs(
     bool enforceChirality = true, bool useExpTorsionAnglePrefs = false,
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0);
+
+struct EmbedParameters {
+  unsigned int maxIterations;
```

Comment:
hmm, looking at the way the AdjustQueryParameters is handled, this doesn't seem to be true. SWIG writes the setters and getters for you. Or did you mean something else?